model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -52,7 +52,7 @@ private fun createCopyrightFinding(statement: String, path: String, line: Int) =
 class FindingsMatcherTest : WordSpec() {
     companion object {
         private const val LICENSE_FILE_PATH = "a/LICENSE"
-        private const val THRESHOLD = 5
+        private const val THRESHOLD = FindingsMatcher.TOLERANCE_LINES_DEFAULT_VALUE + 13
```

Comment:
Why "+ 13" if we had "5" (which is what `TOLERANCE_LINES_DEFAULT_VALUE` is set to) before?

Reply:
The intention is to use a value different to the default value which I think is better than using the exact same value, because if you are using the default value you cannot tell whether the default one was used or the one passed was used.

Reply:
One the other hand, using the default value proves that the choice of the default value is a good one. And the "13" seems to be completely arbitrary here, which is confusing, at least for me.